wsmaster/codenvy-hosted-api-organization/src/main/java/com/codenvy/organization/shared/model/Organization.java

```diff
@@ -0,0 +1,42 @@
+/*
+ *  [2012] - [2016] Codenvy, S.A.
+ *  All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of Codenvy S.A. and its suppliers,
+ * if any.  The intellectual and technical concepts contained
+ * herein are proprietary to Codenvy S.A.
+ * and its suppliers and may be covered by U.S. and Foreign Patents,
+ * patents in process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Codenvy S.A..
+ */
+package com.codenvy.organization.shared.model;
+
+import org.eclipse.che.commons.annotation.Nullable;
+
+/**
+ * Describes group of users that can use common resources
+ *
+ * @author gazarenkov
+ * @author Sergii Leschenko
+ */
+public interface Organization {
+
+    /**
+     * Returns id of organization
+     */
+    String getId();
+
+    /**
+     * Returns name of organization
```

Comment:
It says nothing about name uniqueness i think that the fact that user name and organization name are in the same namespace is important